api_tests/files/serializers/test_file_serializer.py

```diff
@@ -70,11 +70,13 @@ def test_file_serializer(self, file_one, node, node_folder):
 
         # test_date_modified_formats_to_old_format
         req = make_drf_request_with_version(version='2.0')
+        file_one.date_modified = modified_tz_aware  # date_modified is past into the serializer via view logic
         data = FileSerializer(file_one, context={'request': req}).data['data']
-        assert modified_tz_aware == data['attributes']['date_modified']
+        assert modified_tz_aware.strftime('%Y-%m-%dT%H:%M:%S.%f') == data['attributes']['date_modified']
```

Comment:
`VersionedDateTimeField` recognizes this as a not having milliseconds  so the more consistent versioning actually changes the `2.0` behavior slightly.